src/color-palette.less

```diff
@@ -0,0 +1,135 @@
+//   20 | e2efd8  | d4e7e6  | d6e8fa  | d3daeb  | f7e0d9  | fff0dd  | e3e2df  | d2d3d5  |
+//   10 |         |         |         |         |         |         |         | e7e8e9  |
+//    5 |         |         |         |         |         |         |         | f7f8f9  |
+//
+// CFPB Black: 101820
+
+
+// Green family
+@dark-green:   #1e9642;
+@green:        #20aa3f; // Primary brand green color
+@green-80:     #66c368;
+@green-60:     #addc91; // Formerly known as "Green: Midtone"
+@green-40:     #c7e5b3;
+@green-20:     #e2efd8;
+
+// Teal family
+@dark-teal:    #005e5d; // Formerly known as regular ol' "Teal"
+@teal:         #257675; // Close to what was previously called "Teal 80"
+@teal-80:      #579695;
+@teal-60:      #89b6b5;
+@teal-40:      #b4d2d1;
+@teal-20:      #d4e7e6;
+
+// Pacific family
+@dark-pacific: #0050b4;
+@pacific:      #0072ce;
+@pacific-80:   #4497dc;
+@pacific-60:   #7eb7e8;
+@pacific-40:   #afd2f2;
+@pacific-20:   #d6e8fa;
+
+// Navy family
+@dark-navy:    #002d72; // Formerly known as regular ol' "Navy"
+@navy:         #254b87; // Close to what was previously called "Navy 80"
+@navy-80:      #5674a3;
+@navy-60:      #889cc0;
+@navy-40:      #b3c0d9;
+@navy-20:      #d3daeb;
+
+// Red family (formerly known as "Red Orange")
+@dark-red:     #b63014;
+@red:          #d14124;
+@red-80:       #dd735d;
+@red-60:       #e79e8e;
+@red-40:       #f0c3b8;
+@red-20:       #f7e0d9;
+
+// Gold family
+@dark-gold:    #dc731c;
+@gold:         #ff9e1b;
+@gold-80:      #ffb858;
+@gold-60:      #ffce8d;
+@gold-40:      #ffe1b9;
+@gold-20:      #fff0dd;
+
+// Neutral family
+@dark-neutral: #63574e;
+@neutral:      #796e65;
+@neutral-80:   #948b84;
+@neutral-60:   #b5aca5;
+@neutral-40:   #cdc7c2;
+@neutral-20:   #e3e2df;
+
+// Gray family
+@dark-gray:    #43484e;
+@gray:         #5a5d61;
+@gray-80:      #75787b; // Formerly known as regular ol' "Gray"
+@gray-60:      #919395; // Formerly known as "Gray 80"
+@gray-40:      #b4b5b6; // Close to what was known as "Gray 50"
+@gray-20:      #d2d3d5;
+@gray-10:      #e7e8e9; // Close to what was known as "Gray 20"
+@gray-5:       #f7f8f9;
+
+// with special guests
+@black:        #101820; // Also known as "CFPB Black"
+@white:        #fff;
+
+
+/**
+ * When adding this theme to a CFPB project using a version of our color palette
+ * that predates November 13, 2015, you will have some deprecated color
+ * variables to deal with.
+ *
+ * Run the list of project-wide Find and Replace actions given below
+ * to update deprecated variables to their current equivalents.
+ *
+ * IMPORTANT: You must execute these in the specified order
+ * to ensure that you don't overwrite previous updates.
+ *
+ * ALSO IMPORTANT: When running replacements #3, #7, and #19, ensure that you
+ * have the regular expression option turned on in your editor.
+ * Failure to do so will replace the partial match in every tint of that color.
+ */
```

Comment:
This four-paragraph disclaimer is really intimidating but I'm guessing it's rare someone will have to update the colors.